packages/editor/src/components/editor/index.js

```diff
@@ -74,11 +74,11 @@ function Editor( {
 					<EditorInterface { ...props }>
 						{ extraContent }
 					</EditorInterface>
+					{ children }
```

Comment:
`<Sidebar>` is the settings sidebar, which renders the settings toggle in the topbar.

The global styles toggle is rendered through the children, so moving the children to render first should reorder the items.

My main concern though is that it might allow plugins to render toggles after the global styles toggle but before the settings one. I need to do some more testing around that.